debezium-connector-mongodb/src/main/java/io/debezium/connector/mongodb/ConnectionContext.java

```diff
@@ -399,7 +399,7 @@ public void executeBlocking(String desc, BlockingConsumer<MongoClient> operation
         }
 
         private boolean isRunning() {
```

Comment:
`isRunning` method is used only with negation, it may be better to rename it to `isStopped` and delete the negation in the method and places where the method is used.

Reply:
@gunnarmorling @jchipmunk ok, how about now?

Reply:
I like it :)

Reply:
It looks good to me.